### PR TITLE
Fix docstring of ComputeIkParamPosition

### PR DESCRIPTION
**Pull request by felixvd**
This PR removes unused parameters from the docstring. `realtimerobottask3.py` does not do anything with them.

### CODE REVIEW

File: python/mujincontrollerclient/realtimerobotclient.py [hunk 1]
```diff
@@ -822,19 +822,14 @@ def SetRobotBridgeIOVariables(self, iovalues, robotname=None, timeout=10, useweb
...
     def ComputeIkParamPosition(self, name, robotname=None, timeout=10, usewebapi=None, **kwargs):
-        """
+        """Given the name of a Kinbody, computes the manipulator (TCP) position in the kinbody frame to generate values for an IKParameterization.
 
         Args:
-            name (str):
-            robotname (str, optional): Name of the robot
-            timeout (float, optional):  (Default: 10)
+            name (str): Name of the Kinbody (the robot).
+            robotname (str, optional): (unused) Name of the robot.
```
Comment (kanbouchou) [inline python/mujincontrollerclient/realtimerobotclient.py:829]: @felixvd it is a bit hard to follow in the code, but I think ``robotname`` is used.

**Review by kanbouchou — CHANGES_REQUESTED** [review]

File: python/mujincontrollerclient/realtimerobotclient.py (hunk 1 above)
Comment (felixvd) [reply python/mujincontrollerclient/realtimerobotclient.py]: Right. It's redundant, but `SetGoals` uses it to set up the environment.

**Comment by cielavenir** [comment]
oh that docstring was actually that of ComputeIKFromParameters, nice catch :eyes:
